Vagrantfile

```diff
@@ -167,6 +205,9 @@ broadcast_address = n | (~n.instance_variable_get(:@mask_addr) & IPAddr::IN4MASK
 cluster_ip_cidr = settings["pod_network"]["cluster_ip_cidr"]
 service_ip_cidr = settings["pod_network"]["service_ip_cidr"]
 
+calico_env_var = settings["ansible"]["group_vars"]["all"]["calico_config"]["calico_env_var"]
+calico_env_var_value = settings["ansible"]["group_vars"]["all"]["calico_config"]["calico_env_var_calue"]
```

Comment:
Typo in `calico_env_var_calue`

Reply:
I think we should add a check to prevent this kind of issues. I'll open a new one.

Reply:
thank you!!